src/main/java/com/exactpro/th2/common/schema/grpc/router/impl/DefaultStubStorage.java

```diff
@@ -122,6 +122,7 @@ public T getStub(@NotNull Message message, @NotNull AbstractStub.StubFactory<T>
                             .keepAliveTime(clientConfiguration.getKeepAliveInterval(), TimeUnit.SECONDS)
                             .usePlaintext()
                             .intercept(new GrpcInterceptor(service.pinName, methodInvokeCounter, requestBytesCounter, responseBytesCounter))
+                            .maxInboundMessageSize(endpoint.getMaxMessageSize())
```

Comment:
You forget about AbstractGrpcRouter.getOrCreateChannel method.

Reply:
https://github.com/th2-net/th2-common-j/pull/181/files

Reply:
Also, please have a look at the service side in the AbstractGrpcRouter.startServer method. The NettyServerBuilder class has the same maxInboundMessageSize method